tests/session/session-getTransactionOptions-001.phpt

```diff
@@ -0,0 +1,92 @@
+--TEST--
+MongoDB\Driver\Session::getTransactionOptions()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+$session = $manager->startSession();
+
+var_dump($session->getTransactionOptions());
+
+// Ending session until CDRIVER-3366 is fixed
+$session->endSession();
```

Comment:
The issue uncovered a bug where aborting a transaction in the starting state (meaning it hasn’t run any commands yet) did not reset transaction options. Since that issue broke the tests for the original issue in CDRIVER-3366, I’m fixing it there.

This same issue is coming into play here since we’re never running an operation in the session, so the workaround is to end the session and recreate it to ensure we’re not taking any transaction options with us to the next test.

Reply:
Yep, looks like re-assigning the `$session` variable correctly ends the session and transaction. I've removed the calls, except for the `startSession()` which is indeed necessary due to CDRIVER-3366 (see https://github.com/alcaeus/mongo-c-driver/commit/db9c035401d629b352c7439512ae31a8bb6d6de1).